### ISSUE TITLE
Option for change default response

### ISSUE THREAD

**Issue by mjablecnik** [xnbox/DeepfakeHTTP#14]
Hello, I know that when I send request for not existing endpoint so I gain 400 Bad request error response.
But what to do when I want to have some another default response? For example 404 Not found?

Exists some option which can change what DeepfakeHTTP return when the endpoint doesn't exists?

**Comment by xnbox** [comment]
Hi!
I think I will add a command-line option, for custom status code when the request is not found.
Thanks!

P.S.
By the way, I added a special option for JSON soft comparison (like you wanted):

```json
 "customers": {
        "123": {"fname": "John", "lname": "Doe", "email": ["john@example.com", "johndoe@example.com"]},
        "456": {"fname": "Lora", "lname": "Corban", "email": ["lora@example.com", "loracorban@example.com"]},
        "789": {"fname": "Ted", "lname": "Brown", "email": ["tedbrown@example.com"]}
    }
```

```json
 "customers": {
        "456": {"email": ["lora@example.com", "loracorban@example.com"], "fname": "Lora", "lname": "Corban",},
        "123": {"fname": "John", "lname": "Doe", "email": ["john@example.com", "johndoe@example.com"]},
        "789": {"fname": "Ted", "lname": "Brown", "email": ["tedbrown@example.com"]}
    }
```
will be interpreted as equals.
This feature (soft JSON comparison) will be present in the nearest release.

**Comment by mjablecnik** [comment]
Oh thank you very much. ;-)

**Comment by xnbox** [comment]
release v1.0.15 contains the features described above.

**Comment by mjablecnik** [comment]
Are you not using semantic versioning? https://semver.org
MAJOR.MINOR.PATCH for versioning of your application?
If this new things are features and not only bugs so it should be increased MINOR number in your version instead of PATCH.
So new version should be v1.1.0 or not?

**Comment by xnbox** [comment]
I didn't know about that :(
From the next version I will switch to this versioning system.
Thanks for the tip!

**Comment by xnbox** [comment]
implemented in v1.0,15
